packages/math-input/src/store/pager-reducer.ts

```diff
@@ -41,70 +59,41 @@ const pagerReducer = function (
                 dx: 0,
             };
 
-        case "SetPageSize":
+        case SetPageSizeActionType:
             return {
                 ...state,
-                // @ts-expect-error [FEI-5003] - TS2339 - Property 'pageWidthPx' does not exist on type '{ type: string; }'.
                 pageWidthPx: action.pageWidthPx,
             };
 
-        case "PressKey":
-            // @ts-expect-error [FEI-5003] - TS2339 - Property 'key' does not exist on type '{ type: string; }'.
+        case PressKeyActionType:
             const keyConfig = KeyConfigs[action.key];
 
             // Reset the keypad page if the user performs a math operation.
             if (
                 keyConfig.type === KeyTypes.VALUE ||
                 keyConfig.type === KeyTypes.OPERATOR
             ) {
-                return pagerReducer(state, {type: "ResetKeypadPage"});
```

Comment:
Some of these cases were just calling other cases that were only used by the first case. It seemed like needless complexity, so I merged a couple of cases.